src/backend/db/products.js

```diff
@@ -1,30 +1,92 @@
 import { v4 as uuid } from "uuid";
-
+import { productImage } from "../../assets";
 /**
  * Product Database can be added here.
  * You can add products of your wish with different attributes
  * */
 
 export const products = [
+  // {
+  //   _id: uuid(),
+  //   title: "You Can WIN",
+  //   author: "Shiv Khera",
```

Comment:
You can remove the unnecessary commented code

Reply:
Done the changes